app/Http/Requests/CreateProjectRequest.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace App\Http\Requests;
+
+use App\Project;
+
+class CreateProjectRequest extends Request
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return $this->user() != null;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'title' => 'required|unique:projects|regex:/([A-Za-z0-9_ .]+)/|' . $this->minMaxFormatter('title'),
```

Comment:
Regex needs be updated to require a least one letter or else user can name it _________
regex:/(?=.*[a-zA-Z])([A-Za-z0-9_ .]+)/ I believe would work